spacy/errors.py

```diff
@@ -887,6 +887,7 @@ class Errors(metaclass=ErrorsWithCodes):
     E1021 = ("`pos` value \"{pp}\" is not a valid Universal Dependencies tag. "
              "Non-UD tags should use the `tag` property.")
     E1022 = ("Words must be of type str or int, but input is of type '{wtype}'")
+    E1023 = ("A pattern with {ent_id} id is not present in EntityRuler patterns.")
```

Comment:
I would use the format `ID \"{ent_id}\"` instead, in case IDs have spaces or something.

Reply:
Ah, OK!